<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QbsProjectManager::Internal::QbsInstallStepConfigWidget</class>
 <widget class="QWidget" name="QbsProjectManager::Internal::QbsInstallStepConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>146</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="installRootLabel">
     <property name="text">
      <string>Install root:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="flagsLabel">
     <property name="text">
      <string>Flags:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="dryRunCheckBox">
       <property name="text">
        <string>Dry run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="keepGoingCheckBox">
       <property name="text">
        <string>Keep going</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="removeFirstCheckBox">
       <property name="text">
        <string>Remove first</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="commandLineKeyLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Equivalent command line:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="commandLineTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="installRootValueLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dryRunCheckBox</tabstop>
  <tabstop>keepGoingCheckBox</tabstop>
  <tabstop>removeFirstCheckBox</tabstop>
  <tabstop>commandLineTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
